w09b01/pgdp/AITests.java

```diff
@@ -1447,73 +1440,46 @@ public void test59() {
 
         Assertions.assertEquals(0, move.x());
         Assertions.assertEquals(1, move.y());
-        Assertions.assertEquals(7, move.value());
-
     }
 
     @Test
-    @DisplayName("DG Test 61 mehrere gewinne verhindern erster spieler mit diagonale test 1")
-    public void test61() {
+    @DisplayName("DG Test 62 mehrere gewinne verhindern erster spieler mit diagonale test 2")
+    public void test62() {
         this.board = new Field[][]{
-                {null, new Field(0, false), new Field(1, false)},
+                {null, new Field(8, false), new Field(0, false)},
                 {null, new Field(7, false), new Field(8, true)},
-                {null, new Field(7, true), new Field(8, false)}};
+                {new Field(5, true), new Field(7, true), new Field(8, false)}};
         this.firstPlayedMoves = new boolean[]{
                 false, false, false,
                 false, false, false,
                 false, true, true};
         this.secondPlayedMoves = new boolean[]{
-                true, true, false,
+                true, false, false,
                 false, false, false,
-                false, true, true};
+                true, true, true};
 
         Move move = this.ai.makeMove(this.board, true, this.firstPlayedMoves, this.secondPlayedMoves);
 
         Assertions.assertEquals(0, move.x());
         Assertions.assertEquals(0, move.y());
         Assertions.assertEquals(6, move.value());
```

Comment:
```suggestion
        Assertions.assertEquals(5, move.value());
```

Why do you expect a value of 6 if the opponent has already used up his "6"?

Reply:
Also applies to: https://github.com/MaximilianAnzinger/pgdp2223-tests/blob/017a9c57de83513b21a5e3d4ba5993f76316da1b/w09b01/pgdp/AITests.java#L1488

and i don't see why the value can not be 1 in this case: https://github.com/MaximilianAnzinger/pgdp2223-tests/blob/017a9c57de83513b21a5e3d4ba5993f76316da1b/w09b01/pgdp/AITests.java#L1512


> Die Spielsteine der Spieler besitzen ab jetzt die Werte 000 bis 888. Jeder Spieler darf einen Spielstein nun nicht nur auf ein freies Feld legen, sondern auch auf ein Feld, auf den der Gegner einen Spielstein mit einem niedrigeren Wert gelegt hat.

-> if opponent has already placed 6 7 and 8 he can not play another move on a field with value 5

> Wenn der Gegner im nächsten Zug gewinnen kann und deine KI dies durch einen Zug verhindern kann, soll dieser gewählt werden, außer deine KI kann selbst direkt gewinnen.

afaik this rule would not be violated in any of the three cases